recipes/repeatmodeler/run_test.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
+set -x -e
+
+# build the db with BuildDatabase
+BuildDatabase -engine ncbi -name db test.fa 2> /dev/null
+
+# run repeat modeler
+RepeatModeler -engine ncbi -database db 2> /dev/null
+
+# check the result
+if [ -s RM_*/consensi.fa ] ; then 
```

Comment:
You can replace the entire `if` with `test -s RM_*/consensi.fa` (the exit code of the script is the exit code of the last command; also you have `set -e`, so it’ll exit anyway if `test` fails).